compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ReshapeFusion.cpp

```diff
@@ -256,10 +257,13 @@ static std::optional<SmallVector<Value>> fuseAttentionWithReshapeByExpansion(
 
   // Create a new `AttentionOp` that has the computed operands/indexing maps.
   TypeRange resultTypes = ValueRange(outputs).getTypes();
+  std::optional<Value> mask =
+      attentionOp.getMask() ? std::optional<Value>(expandedOpOperands.back())
```

Comment:
do we need to wrap `expandedOpOperands.back()` with `std::optional<Value>(blah)` or can we just set it directly?

Reply:
Nope, it complains if I remove the wrapping

Reply:
cool cool